crates/ra_lsp_server/tests/heavy_tests/main.rs

```diff
@@ -263,20 +263,25 @@ fn main() {}
 #[test]
 fn test_missing_module_code_action_in_json_project() {
     let tmp_dir = TempDir::new().unwrap();
+
+    let path = format!("{}", tmp_dir.path().display());
```

Comment:
I think we don't need to call `display` at all, we can pass an `&Path` to `json!` direclty and it'll figure it out